large-borrows-governance/src/agent.spec.js

```diff
@@ -244,15 +270,17 @@ describe('handleTransaction', () => {
 
   it('returns two findings when a borrow event is emitted from the cCOMP token contract and the address that borrowed COMP exceeds the proposal and vote quorum thresholds', async () => {
     const {
-      minAmountCOMP: proposalMinCOMP, type: proposalType, severity: proposalSeverity,
+      type: proposalType, severity: proposalSeverity,
     } = borrowLevels.proposal;
     const {
-      minAmountCOMP: votingMinCOMP, type: votingQuorumType, severity: votingQuorumSeverity,
+      type: votingQuorumType, severity: votingQuorumSeverity,
     } = borrowLevels.votingQuorum;
```

Comment:
```suggestion
    const {
      proposal: {
        type: proposalType,
        severity: proposalSeverity,
      },
      votingQuorum: {
        type: votingQuorumType,
        severity: votingQuorumSeverity,
      },
    } = borrowLevels;
```

Reply:
fixed in commit fd3e78c72eeba6a7d7018e8bd6685ba2b0dd06c1